Assets/_Project/Code/Managers/categoryDrawManager.cs

```diff
@@ -0,0 +1,85 @@
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using static constants;
+
+public class CategoryDrawManager : MonoBehaviour
+{
+    int currentRound = 0;
+    [SerializeField] Wheel wheel;
+    [SerializeField] TextMeshProUGUI categoryDisplayText;
+    [SerializeField] TextMeshProUGUI roundDisplayText;
```

Comment:
Serializujesz TMP? Poza tym typ tego nie powinien być TMPText?

Reply:
[SerializeField] to właśnie ten trik który mi podesłałeś, a wolałem to sobie przeciągnąć w inspektorze. Mogę to pobierać w kodzie.
Faktycznie, TMP_Text byłoby lepsze. Działało, bo TextMeshProUGUI dziedziczy po TMP_Text więc zostawiłem, bo nie znam tego za bardzo.

Reply:
A boże, myślałem że Serialize to jest do jsona, przepraszam serdecznie